src/main/java/use_cases/add_edit_expenses_use_case/ExpenseUCI.java

```diff
@@ -58,37 +57,36 @@ public ExpenseOD addExpenseInMonth(ExpenseID expenseIDAdd) throws EntityExceptio
             double valueDouble = toDouble(expenseIDAdd.getValue());
 
             Category selectedCategory = findCategory(categoryList, expenseID.getOldCategory());
-            Category other = findCategory(categoryList, "Other");
 
             if (valueDouble < 0) {
-                // 2. Expense budget less than 0 fail
+                // 2. Expense value less than 0 fail
                 ExpenseOD expenseODFailAdd = new ExpenseOD("Expense value can't be less than $0. Please try again!");
                 return expenseOB.fail(expenseODFailAdd);
             }
-            for (Expense expense1 : monthExpenseList) {
```

Comment:
What you're saying makes sense.

Reply:
They're just moved under from lines 82-89

Reply:
Yup, I saw! but instead the should be checked where I specified above, since if they are checked below you are not checking the recurring expenses (you are checking in the recurExpense list which is redundant, so if you move this code block back it works better!).